docs/pages/docs/components/button.mdx

```diff
@@ -92,3 +92,149 @@ These sizes are standardized across form components.
 </Container>
 `}
 />
+
+## Types
+
+<Playground
+  code={`
+<Container row direction={['column', 'row', 'row']} gap={10}>
+  <Container align="center" justify={"center"}>
+    <Button type={"primary"}>Primary</Button>
+  </Container>
+
+  <Container align="center" justify={"center"}>
+    <Button type={"secondary"}>Secondary</Button>
+  </Container>
+
+  <Container align="center" justify={"center"}>
+    <Button type={"danger"}>Danger</Button>
+  </Container>
+  
+  <Container>
+    <Button type={"warning"}>Warning</Button>
+  </Container>
+  
+  <Container>
+    <Button type={"success"}>Success</Button>
+  </Container>
+  
+  <Container>
+    <Button type={"info"}>Info</Button>
+  </Container>
+  
+  <Container>
+    <Button type={"dark"}>Dark</Button>
+  </Container>
+  
+  <Container>
+    <Button type={"light"}>Light</Button>
+  </Container>
+</Container>
+`}
+/>
+
+## Variant-Ghost
+
+<Playground
+  code={`
+<Container row direction={['column', 'row', 'row']} gap={10}>
+  <Container align="center" justify={"center"}>
+    <Button variant={"ghost"} type={"primary"}>
+      Primary
+    </Button>
+  </Container>
+
+  <Container align="center" justify={"center"}>
+    <Button variant={"ghost"} type={"secondary"}>
+      Secondary
+    </Button>
+  </Container>
+
+  <Container align="center" justify={"center"}>
+    <Button variant={"ghost"} type={"danger"}>
+      Danger
+    </Button>
+  </Container>
+  
+  <Container align="center" justify={"center"}>
+    <Button variant={"ghost"} type={"warning"}>
+      Warning
+    </Button>
+  </Container>
+  
+  <Container align="center" justify={"center"}>
+    <Button variant={"ghost"} type={"success"}>
+      Success
+    </Button>
+  </Container>
+  
+  <Container align="center" justify={"center"}>
+    <Button variant={"ghost"} type={"info"}>
+      Info
+    </Button>
+  </Container>
+  
+  <Container align="center" justify={"center"}>
+    <Button variant={"ghost"} type={"dark"}>
+      Dark
+    </Button>
+  </Container>
+  
+  <Container align="center" justify={"center"}>
+    <Button variant={"ghost"} type={"light"}>
+      Light
+    </Button>
+  </Container>
+</Container>
+`}
+/>
+
+## Loading
+
+<Playground
+  code={`
+<Container row direction={['column', 'row', 'row']} gap={10}>
+  <Container align="center" justify={"center"}>
+    <Button size={"small"} loading>
+      Get started
+    </Button>
+  </Container>
+
+  <Container align="center" justify={"center"}>
+    <Button loading>Get started</Button>
+  </Container>
+
+  <Container align="center" justify={"center"}>
+    <Button size={"large"} loading>
+      Get started
+    </Button>
+  </Container>
+</Container>
+`}
+/>
+
+## Disabled
+
+<Playground
+  code={`
+<Container row direction={['column', 'row', 'row']} gap={10}>
+  <Container align="center" justify={"center"}>
+    <Button prefix={"👋"} disabled>
+      Get started
+    </Button>
+  </Container>
+
+  <Container align="center" justify={"center"}>
```

Comment:
Not necessary in this use case, remove the `align` and `justify` props, example:
``<Container align="center" justify={"center"}>`` ➡️ ``<Container>``